libraries/entities/src/EntityItem.cpp

```diff
@@ -3479,6 +3479,9 @@ void EntityItem::addGrab(GrabPointer grab) {
         simulation->addDynamic(action);
         markDirtyFlags(Simulation::DIRTY_MOTION_TYPE);
         simulation->changeEntity(getThisPointer());
+
+        // don't forget to set isMine() for locally-created grabs
+        action->setIsMine(grab->getOwnerID() == Physics::getSessionUUID());
```

Comment:
We were forgetting to set our own local grabs as "isMine" which was causing logic which relied on `EntityItem::stillHasGrabActions()` (now called `EntityItem::stillHasGrabMyAction()` for clearer code) to fail.